carta/session.py

```diff
@@ -388,10 +388,28 @@ def append_image(self, path, hdu="", complex=False, expression=ArithmeticExpress
         complex : {2}
             Whether the image is complex. Set to ``False`` by default.
         expression : {3}
-            Arithmetic expression to use if appending a complex-valued image. The default is :obj:`carta.constants.ArithmeticExpression.AMPLITUDE`.
+            Complex expression to use if appending a complex-valued image. The default is :obj:`carta.constants.ComplexExpression.AMPLITUDE`.
         """
         return Image.new(self, path, hdu, True, complex, expression)
 
+    @validate(String(), String(), String(r"\d*"), Boolean())
+    def load_LEL_image(self, directory, expression, hdu="", append=False):
```

Comment:
I think this should be called `open_LEL_image`, for consistency. Possibly we should rename the hypercube function to `open_stokes_hypercube` as well.

Reply:
But the form `open_something` does not include the append function. The `load_stokes_hypercube` and this `load_LEL_image` allows users to set append to true. That was my understanding, or I just overthought about that?

Reply:
This is addressed in the longer comment I just made -- I think we need to retire `append_image` and add an `append` flag to `open_image`.